tools/build_config/Jenkinsfile

```diff
@@ -78,13 +78,15 @@ pipeline {
               module load gcc/\$GCC_VERSION &&
               module load openmpi/\$OPEN_MPI_VERSION &&
               ./tools/build_config/build.sh --print_versions --build \
-                --cmake_flags "-DHerbert_RELEASE_TYPE=$RELEASE_TYPE"
+                  --cmake_flags "-DHerbert_RELEASE_TYPE=$RELEASE_TYPE" \
+                  --matlab_release \$MATLAB_VERSION
```

Comment:
having this as `_VERSION` is now confusing isn't it; maybe we need to change the Jenkins variable too for consistency

Reply:
Created #94 for this issue